nimble_build_system/cad/fasteners.py

```diff
@@ -121,3 +123,60 @@ def length(self):
         Getter for the length of the screw.
         """
         return self._length
+
+
+class Ziptie(Fastener):
+    """
+    Ziptie fastener, that shows the straight, ununsed condition only.
+    """
+    #pylint: disable=R0913
+
+    _thickness = 1.6  # mm
+    _width = 4  # mm
+    _length = 100  # mm
+    _fastener_type = "ziptie"
+
+
+    def __init__(self,
+                 name:str,
+                 *,
+                 position:tuple[float, float, float]=(0.0, 0.0, 0.0),
+                 explode_translation:tuple[float, float, float]=(0.0, 0.0, 0.0),
+                 size:str,
+                 fastener_type:str,
+                 axis:str,
+                 length:float):
+
+        self._length = length
+        self._width = float(size)
+
+        super().__init__(name,
+                         position=position,
+                         explode_translation=explode_translation,
+                         size=size,
+                         fastener_type=fastener_type,
+                         direction_axis=axis)
```

Comment:
Is there a reason that the parameter is `axis` in `Ziptie` but `directional_axis` in the parent class `Fastener`?

Reply:
I think I was trying to keep the variables separate as they're passed through the constructor and super calls on `Screw` and `Ziptie`.